src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -979,11 +1008,174 @@ void testIsPoAttending() {
     @NullAndEmptySource
     void testIsPoAttending(String value) {
         SscsCaseData caseData = SscsCaseData.builder()
-                .dwpIsOfficerAttending(value)
-                .build();
+            .dwpIsOfficerAttending(value)
+            .build();
         boolean result = HearingsDetailsMapping.isPoAttending(caseData);
 
         assertThat(result).isFalse();
     }
 
+    @DisplayName("When DWP is attending then return Face to Face as preferred hearing type")
+    @Test
+    void whenDwpIsAttending_thenReturnFaceToFace_asPreferredHearingType() {
+        SscsCaseData caseData = SscsCaseData.builder()
+            .dwpIsOfficerAttending(YesNo.YES.getValue())
+            .appeal(Appeal.builder()
+                        .hearingType(HearingChannel.FACE_TO_FACE.getHmcReference())
+                        .build())
+            .build();
+        List<HearingChannel> result = getHearingChannel(caseData);
+        assertEquals(HearingChannel.FACE_TO_FACE, result.get(0));
+    }
+
+    @DisplayName("When one of the parties contains Face to Face select Face to Fact as preferred value")
+    @Test
```

Comment:
You can combine these three tests into one test using parameterised test. You'd pass in face to face, video and telephone YesNoTypes for each test and the expected HearingCHannel

Reply:
Nope, we are passing stuff into the builder and each test should have its own purpose.

Reply:
These tests repeat the exact same (save the parameters) chunk of code 3 times. If you were writing non test code would you write three different methods that are 98% the same? You'd extract it into one and pass what needs to change in. It makes this a lot easier to review and be worked on in the future.